src/main/scala/co/teapot/tempest/algorithm/MonteCarloPPR.scala

```diff
@@ -23,26 +23,41 @@ import scala.concurrent.{Await, Future}
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.util.Random
 
+
 /**
   * Performing Monte Carlo walks from a list of seed nodes (specified by their ids)
   */
 object MonteCarloPPR {
-  def estimatePPR(graph: DirectedGraph, startIds: IndexedSeq[Int], params: MonteCarloPageRankParams,
+  def estimatePPR(graph: DirectedGraph,
+                  startIds: IndexedSeq[Int],
+                  firstStepDirection: EdgeDir,
```

Comment:
Here too, can you write a comment that describes why we have firstStepDirection and lengthConstraint? In particular, it would be useful to describe all the kinds of PPR that this method allows: SALSA type on Twitter, PPR type on Twitter, PPR type on a bipartite graph, etc.